WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import Yosemite
+
+final class CardReaderSettingsConnectedViewModel: CardReaderSettingsPresentedViewModel {
+
+    var shouldShow: CardReaderSettingsTriState
```

Comment:
It seems like this is meant to be mutable only from within the context of this class, right? If it is, would you consider declaring it as a `private(set) var`?

Reply:
Doesn't it inherit that from the protocol CardReaderSettingsPresentedViewModel ?

Reply:
Nope. Conformance to protocols is interesting… You can declare a var in a protocol like this:
```
var aThing: String { get }
```

And then conform to it as ` let aThing: String = “whatnot”`

In this particular case, if the var declared in the protocol is read only, and the class that conforms to the protocol is read-write, conformance is still granted, and the var in the class is mutable. Fun times.

Reply:
LOL. TIL. Will fix.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4108/commits/e2599a34a37b3541fb4cbff1585fcc7dda12a64f